### PR TITLE
Add games base

### PR DESCRIPTION
**Pull request by fernando457829**

### CODE REVIEW

**Review by Doges — APPROVED** [review]
LGTM

File: src/locales/en-US/game.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "challenge": {
+    "title": "{{player}} has challenged you to a game of {{gameName}}",
+    "clickToAccept": "Click the {{emoji}} reaction button below to accept.",
+    "timeoutIn": "You have {{timeout}} seconds to accept it",
+    "timeout": "This {{gameName}} invite has timed out."
+  },
+
+  "games": {}
+}
```
Comment (metehus) [inline src/locales/en-US/game.json:10]: You need to include `game` locales into the LocaleLoader: https://github.com/SwitchbladeBot/switchblade/blob/dev/src/loaders/LocaleLoader.js#L46

File: src/structures/game/Matrix.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    this.rows = rows
+    this.columns = columns
+  }
+
+  populate (callback = () => 0) {
+    for (let i = 0; i < this.rows; i++) {
+      this[i] = []
+
+      for (let j = 0; j < this.columns; j++) {
+        this[i][j] = callback()
```
Comment (metehus) [inline src/structures/game/Matrix.js:14]: is callback the right name? also would be cool to put the location at the function, so we could use like
```js
matrix.populate((i, j) => i * j)
```

**Review by metehus — CHANGES_REQUESTED** [review]

File: src/structures/game/Matrix.js (hunk 2 above)
Comment (fernando457829) [reply src/structures/game/Matrix.js:14]: I forgot to put location on function and about the name, I really don't know other name I can put.

Comment (fernando457829) [reply src/structures/game/Matrix.js:14]: I will keep `callback` name, because `Array.prototype.map` receive `callback` too

**Review by Doges — APPROVED** [review]
LGTM

**Review by Doges — APPROVED** [review]
LGTM

**Review by metehus — APPROVED** [review]
Good luck with conflicts

**Comment by Doges** [comment]
NÃO PORRA

**Pull request merged**